packages/@okta/vuepress-site/docs/release-notes/2020-09-3/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Okta API Products Release Notes
+---
+
+## 2020.09.3
+
+| Change                                                    | Expected in Preview Orgs |
+| --------------------------------------------------------- | ------------------------ |
+| [Bug fixed in 2020.09.3](#bugs-fixed-in-2020-09-3)       | September 24, 2020           |
```

Comment:
(#bugs-fixed-in-2020-09-3)-> (#bug-fixed-in-2020-09-3)

Reply:
Good catch!